features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Create/NewApp.jsx

```diff
@@ -106,8 +106,10 @@ class NewApp extends React.Component {
                 this.props.updateApps();
             })
             .catch((error) => {
-                if (error.message === 'Conflict') {
-                    Alert.error('Application already exists');
+                const { response } = error;
+                if (response && response.body) {
+                    const message = response.body.description || 'Error while creating the application';
```

Comment:
Isn't it better to have application name in the error message?

Reply:
I think the `description` have it

Reply:
message returned from description will include the application name as that is the error message returned from the backend. but however seems like name is not in the state of this component.